### PR TITLE
Success modal implementation

### PR DESCRIPTION
**Pull request by jobliz**
Implementation to solve #45. Screenshot of comment sent modal:
comment_sent

### CODE REVIEW

**Comment by rivasvict** [comment]
This looks good.
Just a couple of things:
image
This font should be $potter-font (not sure about the name of this variable but it makes reference to Harry Potter jeje)

Basically for all titles

File: src/client/app/modules/AcceptanceModal.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      this.$title = this.$modal.find(".modal-title");
+      this.$body  = this.$modal.find(".success-body-text");
+      this.$acceptButton = this.$modal.find(".accept-button");
+    }
+    
+    /**
+     * Sets the title of the modal.
+     */
+    setTitle(text) {
+      this.$title.html(text);
```
Comment (rivasvict) [inline src/client/app/modules/AcceptanceModal.js:21]: @jobliz Can you please make this a part of the constructor?

File: src/client/app/modules/AcceptanceModal.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    
+    /**
+     *Sets the text of the modal.
+     */
+    setText(text) {
+      this.$body.html(text);
```
Comment (rivasvict) [inline src/client/app/modules/AcceptanceModal.js:28]: @jobliz same here

**Review by rivasvict — CHANGES_REQUESTED** [review]
@jobliz This seems very nice, just those little things.

**Comment by jobliz** [comment]
Changed modal title fonts and refactored the constructor as requested.

**Pull request closed without merging**